src/http/server/handlers/log_handler.cr

```diff
@@ -22,12 +22,9 @@ class HTTP::LogHandler
     minutes = elapsed.total_minutes
     return "#{minutes.round(2)}m" if minutes >= 1
 
-    seconds = elapsed.total_seconds
-    return "#{seconds.round(2)}s" if seconds >= 1
-
-    millis = elapsed.total_milliseconds
-    return "#{millis.round(2)}ms" if millis >= 1
-
-    "#{(millis * 1000).round(2)}µs"
+    elapsed.total_seconds.humanize(precision: 2, significant: false) do |magnitude, _|
+      magnitude = Number.prefix_index(magnitude).clamp(-9..0)
+      {magnitude, magnitude == 0 ? "s " : "#{Number.si_prefix(magnitude)}s"}
+    end
```

Comment:
Why does this require so much code?

Reply:
Ooops, that's a mistake. I just copied the part from the benchmark formatter with the alignment specifics.